### ISSUE TITLE
compiler: remove `-opt 0`

### ISSUE THREAD

**Issue by niaow** [tinygo-org/tinygo#1199]
It doesn't actually work right now, so we might as well get rid of or disable it until it does.

**Comment by aykevl** [comment]
Or fix it. I think it could be useful for debugging (even if the firmware is probably too big to run on a MCU).

But yes, I agree. We shouldn't "support" anything that doesn't actually work.

**Comment by kenbell** [comment]
FYI - current status:
```
% tinygo help
...
  -opt string
        optimization level: 0, 1, 2, s, z (default "z")
...

% tinygo build --target bluepill -o bugtest.elf --opt 0 ./src/examples/bugtest
error: unknown optimization level: -opt=0
```

**Comment by aykevl** [comment]
I have a fix here: https://github.com/tinygo-org/tinygo/pull/1689

**Comment by deadprogram** [comment]
This was released with v0.18 so now closing. Thank you!
